src/tools/altinn-fleet-stats/backend/altinn_fleet/scanner.py

```
@@ -0,0 +1,576 @@
+        for k, v in obj.items():
+            child = f"{path}.{k}" if path else k
+            yield (child, _value_kind(v))
+            yield from _enumerate_settings_keys(v, child)
+    # Don't recurse into lists for settings keys
+
+
+def _parse_bpmn(bpmn_path: Path) -> list[dict]:
+    if not bpmn_path.exists():
+        return []
+    try:
+        tree = ET.parse(bpmn_path)
```

Ruff (0.15.12)

[error] Using `xml` to parse untrusted data is known to be vulnerable to XML attacks; use `defusedxml` equivalents

(S314)